src/lib/split-view/split-view-panel/split-view-panel-foundation.ts

```diff
@@ -0,0 +1,689 @@
+import { ICustomElementFoundation } from '@tylertech/forge-core';
+
+import { percentToPixels, safeMin, scaleValue } from '../../core/utils/utils';
+import { SplitViewPanelPosition, SPLIT_VIEW_PANEL_CONSTANTS } from './split-view-panel-constants';
+import { ISplitViewPanelAdapter } from './split-view-panel-adapter';
+import { SplitViewOrientation } from '../split-view/split-view-constants';
+import { ISplitViewBase } from '../core/split-view-base';
+import { parseSize } from '../core/split-view-core-utils';
+
+export interface ISplitViewPanelFoundation extends ISplitViewBase, ICustomElementFoundation {
+  position: SplitViewPanelPosition;
+  size: number | string;
+  min: number;
+  max: number | undefined;
+  label: string;
+  open: boolean;
+  getContentSize(): number;
+  getCollapsibleSize(): number;
+  setContentSize(size: number): void;
+  setOrientation(value: SplitViewOrientation): void;
+  updateAccessibility(): void;
+}
+
+export class SplitViewPanelFoundation implements ISplitViewPanelFoundation {
+  // API
+  private _position: SplitViewPanelPosition = 'default';
+  private _size: number | string = '200';
+  private _min = 0;
+  private _max: number | undefined;
+  private _label = 'Split view panel';
+  private _open = true;
+  private _disabled = false;
+  private _disableClose = false;
+  private _autoClose = false;
+
+  private _autoCloseThreshold = 0;
```

Comment:
Let's add a public property for this so it can be controlled separately by a developer.